migrations/233_pl_course__question_params__add.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE pl_courses ADD COLUMN question_params jsonb NOT NULL DEFAULT '{}'::jsonb;
```

Comment:
Does the course actually need question params? I'm not saying it's wrong, I just want a justification since the question is part of the course already anyway. Also, it should be NULLable.

Reply:
I added this based on the plan that was laid out in #3530. I imagine the justification is of convenience, allowing an instructor to change the default value for some common parameter that they deliberately use in many questions. For example, many of my questions choose random scenarios from JSON files of data. Those questions could be programmed to read a JSON filename from params and load scenarios from that file; then, when moving to a new semester, I could add new files for that semester and change the course parameter in order to cycle out old scenarios.

A stretch, perhaps (if I'm already adding a "fall2011.json" file to each of those questions, why not just change the server.py default value while I'm at it?), but I'm sure there are other uses.

Reply:
That makes sense, though your example makes a good use-case for course instance, not course. Anyway, using your examples, I'd suggest adding in the documentation an example where a set of questions may be copied between courses, and keeping the source may make these questions easier to migrate.